build.sbt

```diff
@@ -102,6 +102,35 @@ lazy val coreNative = core.native
     ScalafixPlugin // for some reason `ThisBuild / scalafixScalaBinaryVersion := CrossVersion.binaryScalaVersion(scalaVersion.value)` isn't enough
   )
 
+lazy val experimental = crossProject(JSPlatform, JVMPlatform, NativePlatform)
+  .in(file("experimental"))
+  .dependsOn(core)
+  .settings(stdSettings("zio-prelude-experimental"))
+  .settings(crossProjectSettings)
+  .settings(buildInfoSettings("zio.prelude.experimental"))
+
+lazy val experimentalJVM    = core.jvm
```

Comment:
I believe `experimentalJVM`, `experimentalJS` and `experimentalNative` should be aggregated in `root`.
Also, `experimentalJVM/test` should be added to the `testJVM` alias and analogously for `*JS` and `*Native`.

Reply:
@sideeffffect Fixed now!